include/esp-stub-lib/trax_mem.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#define ERI_DEBUG_OFFSET        0x100000
```

Comment:
Not sure that we need to expose such low-level Xtensa specific things outside the lib. 
Will they be used directly by the stub? 

What do you think about exposing higher level API to work with `apptrace` control block for either Xtensa or RISCV chips? The purpose of control memory block is to provide control over apptrace data transfer.
- For Xtensa control block is a set of TRAX registers. For apptrace TRAX regs do not do any HW operations. They are used just to expose control bits between the host and target. Works like a trivial `data containers` accessible from both sides.  
- For RISCV control block is a struct in memory with the special fields which do the same thing as TRAX regs for Xtensa.

Reply:
> Will they be used directly by the stub?

Ok. I see stub use them.

Reply:
We could expose an API like `esp_stub_lib_ctrl_reg_read` and `esp_stub_lib_ctrl_reg_write`, but the apptrace design is specific to OpenOCD and not intended for general use. So, I preferred to expose only the register_read/write APIs. OpenOCD can do whatever it wants with them.

Reply:
I am ok to have them like this. But if nobody use it except for OpenOCD. Does it make big sense to keep them here? Just a question...

Reply:
This is more like a design decision. While setting up the base structure, my initial strategy was to keep the hardware related definitions in `esp-stub-lib`  same as we avoid calling ROM functions directly from the stub code. We can change this later, but for now let’s leave it here.